### PR TITLE
ANW-961: add breadcrumbs to digital materials listing

### PR DESCRIPTION
**Pull request by donaldjosephsmith**
Checks for breadcrumbs in any linked instances for a digital object and displays them in the digital materials listing

### CODE REVIEW

**Comment by donaldjosephsmith** [comment]
A good point was made that grabbing the full record every time to resolve the linked_instances will be bad for performance. This should be reworked to ideally index (if not already) and get the data from Solr.

File: public/app/views/shared/_result.html.erb [hunk 1]
```diff
@@ -63,10 +63,24 @@
...
         <strong><%= t('context') %>: </strong>
         <span  class="repo_name">
+          <%= badge_for_type('repository') %>
           <%= link_to result.resolved_repository.fetch('name'),
                      app_prefix(repository_base_url(
                       "uri" => result.resolved_repository.fetch('uri'),
                       "slug" => result.resolved_repository.fetch('slug') {|s| nil })) %>
+          <% if result.is_a?(DigitalObject) %>
+            <% resolved_instances_record = 
+                 ArchivesSpaceClient.instance.get_record(result.uri, {'resolve[]' => ['linked_instance_uris:id']}) %>
```
Comment (brianzelip) [inline public/app/views/shared/_result.html.erb:73]: Do we want to make an http request to fetch template data from the view layer, or send the data to the template from a controller?

**Review by brianzelip — COMMENTED** [review]
My review has lead to a draft PR against this PR with suggested changes that come from last week's standup where we identified better handling the use case when a digital object has multiple linked instances.

https://github.com/archivesspace/archivesspace/pull/2875

The draft PR isn't finalized yet -- there's some cleanup to be done -- but it shows handling of multiple linked instances.

**Comment by donaldjosephsmith** [comment]
superseded by #2877

**Pull request closed without merging**